telegraf/map.jinja

```diff
@@ -58,7 +58,7 @@
 {%-   endif %}
 {%- elif grains.pythonversion[0] == 3 %}
 {%-   if osfinger in ['CentOS Linux-8'] %}
-{%-     do telegraf.update({'toml_pkg': 'python3-toml'}) %}
+{%-     do telegraf.update({'toml_pkg': 'python3-pytoml'}) %}
 {%-   elif osfinger in ['CentOS Linux-7'] %}
```

Comment:
@baby-gnu We can just use the `else`:

```suggestion
{%-   if osfinger in ['CentOS Linux-7'] %}
```

Reply:
Thanks. This could be improved later based on `python.jinja` from `libvirt-template`